build_cmake/CMakeLists.txt

```diff
@@ -24,7 +24,7 @@ set(CMAKE_CXX_STANDARD 11)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
 if (NOT TARGET Framework)
-	add_subdirectory(../../Framework/build_cmake ${CodeGen_SOURCE_DIR}/../../Framework/build_cmake/build)
+	add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/../../Framework/build_cmake ${CMAKE_CURRENT_SOURCE_DIR}/../../Framework/build_cmake/build)
```

Comment:
This still won't work. It needs to point to `Framework/build_cmake/Framework`. 

Also, I was wondering about the binary directory specification (the second parameter). In the Framework project, some of the binary directories are built using something like `build_${CMAKE_SYSTEM_NAME}` which will create directories called `build_Windows`, `build_Android`, etc. I guess the point is to allow incremental builds when you work on a platform and then the other. I think it would make sense to have some kind of "standard" for binary directory names, one that would include the system name and the architecture (ex.: build_Android_arm-v8a). Anyways, not a big deal for now, we can figure this out later.